str command
---
bool done
str[] reachable_urgent
str[] reachable_corridors
